components/script/script_thread.rs

```diff
@@ -480,6 +481,9 @@ pub struct ScriptThread {
 
     microtask_queue: MicrotaskQueue,
 
+    /// a vector of MutationObserver objects
```

Comment:
This comment does not give the reader any new information. How about `/// The unit of related similar-origin browsing contexts' list of MutationObserver objects.` instead?